build

```diff
@@ -103,3 +103,12 @@ fi
 
 progress "saving image"
 img save -s /scratch/state -o image/image.tar $REPOSITORY:$TAG
+
+if [ -n "$UNPACK_ROOTFS" ]; then
+  progress "unpacking rootfs"
+  imageConfig=$(tar -Oxf image/image.tar manifest.json | jq -r '.[0].Config')
+  tar -Oxf image/image.tar $imageConfig | jq '{"user":.config.User,"env":.config.Env}' > image/metadata.json
+  RUNC_PATH=$(echo /tmp/img-runc*) # Can be removed once this is fixed https://github.com/genuinetools/img/issues/233
+  PATH=$RUNC_PATH:$PATH img unpack -s /scratch/state -o /scratch/rootfs $REPOSITORY:$TAG
+  rsync -a /scratch/rootfs/ ./image/rootfs
```

Comment:
This all looked a bit funny but https://github.com/concourse/builder-task/issues/24#issuecomment-492139121 explains it. :+1: